src/components/SignIn.tsx

```diff
@@ -268,8 +268,8 @@ const SignIn = ({ onWeb3ProviderSet, onWeb3AuthInstanceSet }: SignInProps) => {
       });
 
       web3AuthInstance.on(ADAPTER_EVENTS.DISCONNECTED, () => {
-        onWeb3ProviderSet(null);
-        setIsSigningIn(false);
+        // onWeb3ProviderSet(null);
+        // setIsSigningIn(false);
```

Comment:
If we aren't using these lines anymore, shouldn't we just remove them?

Reply:
yeah sure thought we would keep it for a while so that we would know if something else comes up